libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -314,7 +314,11 @@ from the FreeRTOSIPConfig.h configuration header file. */
 #endif
 
 #ifndef ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND
-	#define ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND 1
+	#define ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND	1
+#endif
+
+#ifndef ipconfigUDP_PASS_ZERO_CHECKSUM_PACKETS
+	#define ipconfigUDP_PASS_ZERO_CHECKSUM_PACKETS	0
```

Comment:
Same comment as above, where is this documented?  Why would I want to define `ipconfigUDP_PASS_ZERO_CHECKSUM_PACKETS==1`?

Reply:
As you know: an UDP checksum has a value between `0x0001` and `0xFFFF`.
If the calculated checksum is 0, `0xFFFF` will be used in stead.
A value of `0x0000` means: no checksum. It is up to the receiver to accept it or not.
I guess that in the old days, it was a way to increase network performance.